Zombie/ZombieDriver.js

```diff
@@ -1,10 +1,10 @@
 "use strict";
 
-process.on('uncaughtException', e => console.error('Uncaught Exception: ' + e));
-
 const Zombie = require('zombie');
 const zombie = new Zombie();
 
+process.on('uncaughtException', e => console.error('Uncaught Exception: ' + e));
```

Comment:
Can we really move this to the bottom of the file since we're returning the `function(data,callback)`?
Even if we modify the function declaration as such:
````
var bridge = function(data, callback) {
  callback(null, new Bridge(zombie));
}

process.on('uncaughtException', e => console.error('Uncaught Exception: ' + e));
return bridge;
````

It's still ultimately the same result... Right?